ci/ray_ci/container.py

```diff
@@ -0,0 +1,84 @@
+import os
+import subprocess
+
+from typing import List
+
+from ci.ray_ci.utils import chunk_into_n
+
+DOCKER_ECR = "029272617770.dkr.ecr.us-west-2.amazonaws.com"
+DOCKER_REPO = "ci_base_images"
+DOCKER_TAG = f"oss-ci-build_{os.environ.get('BUILDKITE_COMMIT')}"
+
+
+def run_tests(
+    test_targets: List[str],
+    pre_run_commands: List[str],
+    parallelism,
+) -> int:
+    """
+    Run tests parallelly in docker
+    """
+    chunks = chunk_into_n(test_targets, parallelism)
+    runs = [_run_tests_in_docker(chunk, pre_run_commands) for chunk in chunks]
+    exits = [run.wait() for run in runs]
+    return 0 if all(exit == 0 for exit in exits) else 1
+
+
+def _run_tests_in_docker(
+    test_targets: List[str],
+    pre_test_commands: List[str],
+) -> subprocess.Popen:
+    bazel_command = ["bazel", "test", "--config=ci"] + test_targets
+    command = "\n".join(pre_test_commands + [" ".join(bazel_command)])
+    return subprocess.Popen(_get_docker_run_command() + ["/bin/bash", "-ice", command])
```

Comment:
can we unify this with the `run_command()`, maybe have a `_docker_run_bash_script(script)` that returns `_get_docker_run_command() + ["/bin/bash", "-ice", script]`?

Reply:
totally